spec/mercure.md

```diff
@@ -388,8 +406,43 @@ To receive updates marked as `private`, the JWS presented by the subscriber **MU
 claim named `mercure` with a key named `subscribe` that contains an array of topic selectors. See
 (#topic-selectors).
 
-The hub **MUST** check that at least one topic of the update to dispatch matches at least one topic
-selector provided in `mercure.subscribe`.
+The hub **MUST** check that at least one topic of the update to dispatch (*canonical* or
+*alternate*) matches at least one topic selector provided in `mercure.subscribe`.
+
+This behavior makes it possible to subscribe to several topics using URI templates while
+guaranteeing that only authorized subscribers will receive updates marked as private (even if their
+canonical topics are matched by these templates).
+
+Let's say that a subscriber wants to receive updates concerning all *book* resources it has access
+to. The subscriber can use the topic selector `https://example.com/books/{id}` as value of the
+`topic` query parameter. Adding this same URI template to the `mercure.publisher` claim of the JWS
```

Comment:
Don't you mean `mercure.subscribe` here?

Reply:
Yes! Thanks.